### PR TITLE
add a sentance to explain how to migrate Eigen dependency

### PR DESCRIPTION
**Pull request by tfoote**

### CODE REVIEW

**Comment by wjwwood** [comment]
You could just link to the usage of `cmake_modules`:

https://github.com/ros/cmake_modules/blob/0.3-devel/README.md#usage

File: doc/adv_user_guide/catkin_migration_indigo.rst [hunk 1]
```diff
@@ -22,3 +22,7 @@ Removed Eigen CMake config file
 The previously provided CMake config file for Eigen has been removed.
 Instead you might want to use the CMake module provided by the
 `cmake_modules <http://wiki.ros.org/cmake_modules>`_ package.
+
+When up update you must find_package cmake_modules before find_package Eigen.
```
Comment (dirk-thomas) [inline doc/adv_user_guide/catkin_migration_indigo.rst:26]: "When up update"?

Comment (dirk-thomas) [inline doc/adv_user_guide/catkin_migration_indigo.rst:26]: Please use double backticks for code like ``find_package()`.

**Comment by tfoote** [comment]
Good point. I just augmented the cmake_modules documentation https://github.com/ros/cmake_modules/pull/14

**Pull request closed without merging**